flexget/plugins/clients/transmission.py

```diff
@@ -286,7 +286,7 @@ class PluginTransmission(TransmissionBase):
                     'password': {'type': 'string'},
                     'action': {
                         'type': 'string',
-                        'enum': ['add', 'remove', 'purge', 'pause', 'resume'],
+                        'enum': ['add', 'remove', 'purge', 'pause', 'resume', 'start bypass_queue'],
```

Comment:
Was that what you meant?

Reply:
I think more like this:
`'enum': ['add', 'remove', 'purge', 'pause', 'resume', 'bypass_queue']`

Reply:
transmission uses ``Resume Now`` in the context menu, that's why I came up with ``start now``.
On second thought maybe we should use ``resume now`` so it's consistent with the gui?
My apprehension is that with ``bypass_queue`` it's not really clear what the action does

Reply:
I think that bypass_queue is clear enough and also the official term that the application uses, so its best to use it

Reply:
It's just that only from ``bypass_queue`` it's imho not clear what the action is doing.
Maybe ``resume (bypass_queue)``?

Reply:
That's why plugins are documented 👍